src/contractTest/java/uk/gov/hmcts/reform/unspec/BaseRpaTest.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.hmcts.reform.unspec;
+
+import au.com.dius.pact.consumer.ConsumerPactBuilder;
+import au.com.dius.pact.consumer.PactTestRun;
+import au.com.dius.pact.consumer.PactVerificationResult;
+import au.com.dius.pact.model.MockProviderConfig;
+import au.com.dius.pact.model.RequestResponsePact;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.SpecVersion;
+import com.networknt.schema.ValidationMessage;
+import io.restassured.RestAssured;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Paths;
+import java.util.Set;
+
+import static au.com.dius.pact.consumer.ConsumerPactRunnerKt.runConsumerTest;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Slf4j
+public abstract class BaseRpaTest {
+
+    protected static final String PATH = "/fake-endpoint";
+    protected static final String CONSUMER = "unspec_service";
+    protected static final String PROVIDER = "rpa_api";
+
+    protected ObjectMapper objectMapper = new ObjectMapper();
+
+    protected RequestResponsePact preparePact(int statusCode, String description, String body) {
+        // @formatter:off
```

Comment:
what's this comment doing?

Reply:
this will disable the formating controlled by the IDE, as IDE messes up with the indentation of semantics we need there.